### ISSUE TITLE
Question on hyperparameter setting

### ISSUE THREAD

**Issue by Huangzisu** [nusnlp/FSPO#1]
Hi,

Thank you for your excellent open-sourced work! 

I have one question regarding the hyperparameter settings in [main_grpo_qwen_instruct.sh](https://github.com/nusnlp/FSPO/blob/master/main_grpo_qwen_instruct.sh):
```
data.train_batch_size=4
actor_rollout_ref.actor.ppo_mini_batch_size=1024
actor_rollout_ref.rollout.n=8
```
As I understand it, this setup would generate only 32 samples per iteration, which seems much smaller than the required 1024 samples for PPO updates. I might be misunderstanding how these parameters are meant to interact, and I’d really appreciate your response to help me clear up this confusion.

Thank you!

**Comment by turboLJY** [comment]
Hi, the mini batch size is the actual batch size for optimizing the model parameters. So here it uses accumulative gradient descent (1024 / (4*8) = 32). Maybe you can refer to the verl document for more details of these hyper-parameters. Thank you.

**Comment by Huangzisu** [comment]
Thanks for your explanation! It seems that the use of hyperparameter `data.train_batch_size` has changed in newer version of verl, which indicate `Batch size sampled for one training iteration of different RL algorithms.`. This requires `data.train_batch_size >= actor_rollout_ref.actor.ppo_mini_batch_size`, as indicated in [this line](https://github.com/volcengine/verl/blob/578501e2f8a6b42ee0861b58b641340b72dda10c/verl/trainer/ppo/ray_trainer.py#L465) in verl repo.

Thanks again for clarifying this.

**Comment by turboLJY** [comment]
Thank you for telling us. It seems that the meaning of data.train_batch_size has changed in the new version. Thank you again.
